crates/provider-sdk/src/lib.rs

```diff
@@ -104,6 +104,17 @@ pub struct Context {
     pub tracing: HashMap<String, String>,
 }
 
+impl Context {
+    /// Get link name from the request.
+    ///
+    /// While link name should in theory *always* be present, it is not natively included in [`Context`] yet,
+    /// so we must retrieve it from headers on the request.
+    #[must_use]
+    pub fn link_name(&self) -> Option<&String> {
```

Comment:
I think it is more idiomatic to return an `&str` instead

Reply:
Reasonable! Will change this

- [ ] Return `&str` for `Context::link_name`